exonum-java-binding-common/src/main/java/com/exonum/binding/common/proofs/list/ListProofHashNode.java

```diff
@@ -23,19 +23,19 @@
 /**
  * Represents a hash of a Merkle sub-tree: a leaf node in proof trees.
  */
-public final class HashNode implements ListProof {
+public final class ListProofHashNode implements ListProof {
```

Comment:
I am conflicted here: I do prefer the previous conscise name, but admit that this is more informative.

Reply:
Renamed for consistency - HashNode -> ListProofHashNode, ProofListElement -> ListProofElement, will rename ListProof -> ListProofNode in next PR